interfaces/edit_volunteer_cli.py

```diff
@@ -0,0 +1,84 @@
+import controller.volunteer_controller as vc
+from models.camp import Camp
+from interfaces.admin_cli import ManageVolunteerMenu
+
+
+###---- Manage Volunteer Menu ----
+class EditVolunteerMenu(ManageVolunteerMenu):
```

Comment:
Since we are making a sub-menu for editing volunteer profile, it may make more sense to ask for the volunteer username once, save it in the menu instance, so that the user don't need to input the name every time, similar to how we manage login.
Optional enhancement though.

Reply:
Same thought and I've already adjusted the user flow in the new commit. Thanks for the suggestion.